### PR TITLE
fix: key register op must have a change output

### PR DESCRIPTION
**Pull request by obycode**
Changed some `test_debug!` logs `debug!`. These would have helped to debug a VRF key registration error.

### CODE REVIEW

**Comment by obycode** [comment]
Should probably still add a test for this.

**Review by jferrant — APPROVED** [review]

File: stackslib/src/chainstate/burn/operations/leader_key_register.rs [hunk 1]
```diff
@@ -135,33 +135,23 @@ impl LeaderKeyRegisterOp {
         let num_inputs = tx.num_signers();
         let num_outputs = tx.num_recipients();
 
-        if num_inputs == 0 {
-            test_debug!(
+        if num_inputs == 0 || num_outputs < 1 {
```
Comment (jcnelson) [inline stackslib/src/chainstate/burn/operations/leader_key_register.rs:138]: Why not just have `num_inputs == 0`?

Comment (obycode) [reply stackslib/src/chainstate/burn/operations/leader_key_register.rs:138]: You mean num_outputs == 0? This was just copying the existing condition.

Comment (jcnelson) [reply stackslib/src/chainstate/burn/operations/leader_key_register.rs:138]: oh sorry I misread

**Review by jcnelson — APPROVED** [review]

**Comment by obycode** [comment]
I'll go ahead and get this merged. @janniks you can open a new PR for tests if you are able to get to them. Thanks!

**Pull request merged**